test/integration/scheduler/volume_binding_test.go

```diff
@@ -367,6 +371,41 @@ func setupCluster(t *testing.T, nsName string, numberOfNodes int) *testConfig {
 	}
 	go ctrl.Run(controllerCh)
 
+	// Start scheduler
+	configurator := factory.NewConfigFactory(
```

Comment:
By doing this, you are creating another scheduler in the test cluster. `initTestScheduler` at line 348 creates the first one. I guess this has happened by a rebase after a recent change in our integration test library.
I think you should change `createConfiguratorWithPodInformer` in integration/util.go and leave the test as it was.

Reply:
Yes, please make this change here: https://github.com/kubernetes/kubernetes/blob/master/test/integration/scheduler/util.go#L66-L88